zip-0320.rst

```diff
@@ -5,7 +5,357 @@
+  it.” This can be used as a prompt to upgrade the wallet to a version (or
+  alternative) that does support that feature.
+
+  For example: numerous wallets have already upgraded to being able to parse
+  Unified Addresses. Those wallets, on seeing a Traceable Unified Address from
+  Binance, will report to their users that the address is a valid Zcash
+  address, but not yet supported by the wallet. Instead of a user thinking that
+  Binance has made some error, they can contact the wallet’s developer and ask
+  that the wallet be updated.
+
+Cons to Alternative 2
+---------------------
+
+- Unified Address encoding is slightly more complex than the proposed TEX
+  address encoding, and requires use of the F4Jumble encoding algorithm
+  [#F4Jumble]_. However, this can be readily mitigated by providing a
+  purpose-built library for Traceable Unified Address encoding to Producers.
+- A Traceable Unified Address is somewhat longer than a TEX address, although
+  not excessively so.
+
+.. [#BCP14] `Information on BCP 14 — "RFC 2119: Key words for use in RFCs to Indicate Requirement Levels" and "RFC 8174: Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words" <https://www.rfc-editor.org/info/bcp14>`_
+.. [#binance-delisting] `Zcash Community Forum thread "Important: Potential Binance Delisting" <https://forum.zcashcommunity.com/t/important-potential-binance-delisting/45954>`_
+.. [#hanh-profile] 'Zcash Community Forum user @hanh <https://forum.zcashcommunity.com/u/hanh/summary>'_
+.. [#hanh-suggestion] 'Ywallet developer @hanh's proposal <https://forum.zcashcommunity.com/t/important-potential-binance-delisting/45954/112>'_
+.. [#zip-0316-terminology] `ZIP 316: Unified Addresses and Unified Viewing Keys — Terminology <zip-0316#terminology>`_
+.. [#zip-0316-unified-addresses] `ZIP 316: Unified Addresses and Unified Viewing Keys — Encoding of Unified Addresses <zip-0316#encoding-of-unified-addresses>`_
+.. [#zip-0316-unified-requirements] `ZIP 316: Unified Addresses and Unified Viewing Keys — Requirements for both Unified Addresses and Unified Viewing Keys <zip-0316#requirements-for-both-unified-addresses-and-unified-viewing-keys>`_
+.. [#zip-0316-address-expiry] `ZIP 316: Unified Addresses and Unified Viewing Keys — Address Expiration Metadata <zip-0316#address-expiration-metadata>`_
+.. [#protocol-networks] `Zcash Protocol Specification, Version 2023.4.0. Section 3.12: Mainnet and Testnet <protocol/protocol.pdf#networks>`_
+.. [#protocol-transparentaddrencoding] `Zcash Protocol Specification, Version 2023.4.0. Section 5.6.1.1 Transparent Addresses <protocol/protocol.pdf#transparentaddrencoding>`_
+.. [#binance-address-expiry] `Zcash Community Forum post describing motivations for address expiry <https://forum.zcashcommunity.com/t/unified-address-expiration/46564/6>`_
+.. [#Base58Check] `Base58Check encoding — Bitcoin Wiki <https://en.bitcoin.it/wiki/Base58Check_encoding>`_
+.. [#F4Jumble] `ZIP 316: F4Jumble encoding <zip-0316#jumbling>`_
```

Comment:
```suggestion
.. [#F4Jumble] `ZIP 316: Unified Addresses and Unified Viewing Keys — Jumbling <zip-0316#jumbling>`_
```
This could be put next to the other ZIP 316 references and named `[#zip-0316-jumbling]` or `[#zip-0316-f4jumble]` (non-blocking).